BuildingSync.xsd

```diff
@@ -5929,6 +5929,17 @@
           </xs:sequence>
         </xs:complexType>
       </xs:element>
+      <xs:element name="AuditFilingStatus" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>The status of audit filing.</xs:documentation>
```

Comment:
Maybe rephrase: `The status of an audit filing, used to clarify whether or not this audit report is an initial submission (Initial filing) or an amendment to a previously submitted report (Amended filing).`